ega-charts/localega/templates/ingest-deploy.yaml

```diff
@@ -23,6 +23,18 @@ spec:
         role: ingest
         release: {{ .Release.Name }}
     spec:
+{{ if or .Values.ingest.coLocateInbox (not .Values.persistence.enabled) }}
+      affinity:
```

Comment:
Bear in mind that affinity by itself will not prevent other pods from landing in that machine. What happens if somebody accidentally deletes the label, or the node goes down? Will they be re-colocated together in a different node?

Reply:
It will force the ingest pod to be scheduled on the same node as the inbox.
Probably needs to add the same to the inbox in case that one goes down and the ingest pod is still alive.

Reply:
Definitely, that is a scenario where such problem might arise